### PR TITLE
More various ace arsenal fixes

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Fix case issue in verifyLoadout
- Remove uneeded lines related to the perf profiler (doesn't need them anymore)
- Move that one line this PR was about where it should be;
- Fix the ace arsenal camera weird behavior underwater or around rocks, the camera will now clip through rocks or other objects to avoid those issues.
- Gunbags will keep their weapons if you switch to another backpack then back to a gunbag in ace arsenal.

### CODE REVIEW

**Comment by PabstMirror** [comment]
I don't see why this is necessary,
compileStats already does `if (!isNil QGVAR(statsListLeftPanel)) exitWith {};`
so it should have been safe to call at anytime

**Comment by alganthe** [comment]
Nearly forgot about this, since it ended up being a single line I added a bug fix with it :D

**Comment by PabstMirror** [comment]
`findIf`?

**Comment by alganthe** [comment]
new command added in dev branch: https://community.bistudio.com/wiki/findIf

**Comment by jonpas** [comment]
Is it on RC? No merge if it isn't.

**Comment by alganthe** [comment]
it was added before the RC, I didn't check if it was on RC and assumed it was, good point.
I'll check.

**Comment by alganthe** [comment]
Just checked, I can confirm findIf is present on the current RC build (1.82.144616)

**Comment by dedmen** [comment]
why use `findIf` if it's slower than `in` and everything is already built to be lowercase and work fine with `in`?

**Comment by alganthe** [comment]
because, like I said, findIf is case insensitive while in is case sensitive.

**Comment by alganthe** [comment]
There, included all the fixes for the issues i'm aware off.

File: addons/gunbag/XEH_preInit.sqf [hunk 1]
```diff
@@ -24,4 +24,22 @@ PREP_RECOMPILE_END;
     }, _this] call CBA_fnc_execNextFrame;
 }] call CBA_fnc_addClassEventHandler;
 
+[QEGVAR(arsenal,displayOpened), {
+
+    private _center = EGVAR(arsenal,center);
+
+    if (_center call FUNC(hasGunBag)) then {
+        GVAR(arsenalCache) = (backpackContainer _center) getVariable [QGVAR(gunbagWeapon), []];
```
Comment (dedmen) [inline addons/gunbag/XEH_preInit.sqf]: Maybe use a more descriptive varname. As this always stores the gunbag. Why not gunbagCache?
You might want to use arsenalCache for something like an actual arsenalCache later on.

Comment (alganthe) [reply addons/gunbag/XEH_preInit.sqf]: the full var is:
ace_gunbag_arsenalCache, I don't really know if there's a more descriptive varname :/

Comment (dedmen) [reply addons/gunbag/XEH_preInit.sqf]: ooooohhhh. duh. Should've looked at what file that is. Expected it to be in arsenal

**Pull request merged**